src/Sylius/Behat/Context/Ui/EmailContext.php

```diff
@@ -28,22 +29,40 @@ final class EmailContext implements Context
     /** @var EmailCheckerInterface */
     private $emailChecker;
 
-    public function __construct(SharedStorageInterface $sharedStorage, EmailCheckerInterface $emailChecker)
-    {
+    /** @var TranslatorInterface */
+    private $translator;
+
+    public function __construct(
+        SharedStorageInterface $sharedStorage,
+        EmailCheckerInterface $emailChecker,
+        TranslatorInterface $translator
+    ) {
         $this->sharedStorage = $sharedStorage;
         $this->emailChecker = $emailChecker;
+        $this->translator = $translator;
     }
 
     /**
      * @Then it should be sent to :recipient
-     * @Then the email with reset token should be sent to :recipient
      * @Then the email with contact request should be sent to :recipient
      */
     public function anEmailShouldBeSentTo(string $recipient): void
     {
         Assert::true($this->emailChecker->hasRecipient($recipient));
     }
 
+    /**
+     * @Then an email with reset token should be sent to :recipient
+     * @Then an email with reset token should be sent to :recipient in :localeCode locale
+     */
+    public function anEmailWithResetTokenShouldBeSentTo(string $recipient, string $localeCode = 'en_US'): void
+    {
+        $this->assertEmailContainsMessageTo(
+            $this->translator->trans('sylius.email.password_reset.reset_your_password', [], null, $localeCode),
```

Comment:
I'm in favor of this solution, but @pamil had some second thoughts about it.

Reply:
Don't have any better ideas though :)